src/App.tsx

```diff
@@ -13,17 +13,68 @@ interface State {
 
 class App extends Component<Props, State> {
 	public render() {
-		window.location.href = "#startseite";
+
 		return (
-			<div className="App">
-			<SiteLayout />
-			<NavBar />
-			<ProofEditor />
-			</div>
+			<BrowserRouter>
+				<React.Fragment>
+					<Navigation />
+					<Switch>
+						<Route path="/" component={Startseite} exact />
+						<Route path="/aussagenlogisches_beweise" component={ProofEditor} exact />
+						<Route path="/einstellungen" component={Einstellungen} exact />
+						<Route path="/kontakt" component={Kontakt} exact />
+						<Route path="/impressum" component={Impressum} exact />
+						<Route component={ErrorPage404} />
+					</Switch>
+				</React.Fragment>
+			</BrowserRouter>
 		);
 
 	}
 
 }
 
+// These are just demonstration pages. Please don't implement the pages here.
+// Implement them as external components which will be imported
+
+const Startseite = () => {
+	return (
+		<div>
+			<p>Startseite</p>
+		</div>
+	);
+};
+
+const Einstellungen = () => {
+	return (
+		<div>
+			<p>Einstellungen</p>
+		</div>
+	);
+};
+
+const Kontakt = () => {
+	return (
+		<div>
+			<p>Kontakt</p>
+		</div>
+	);
+};
+
+const Impressum = () => {
+	return (
+		<div>
+			<p>Impressum</p>
+		</div>
+	);
+};
+
+const ErrorPage404 = () => {
+	return (
+		<div>
+			<p>404 Page Not Found</p>
+		</div>
+	);
+};
```

Comment:
These should all become their own proper components in separate modules (files)

Reply:
I put them in the file for now to avoid putting more to the folder structur which might be changed later. But this should definitly be the case later.